google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -726,12 +726,10 @@ public ReadChannel reader(BlobSourceOption... options) {
    * <pre>{@code
    * byte[] content = "Hello, World!".getBytes(UTF_8);
    * try (WriteChannel writer = blob.writer()) {
-   *   try {
    *     writer.write(ByteBuffer.wrap(content, 0, content.length));
-   *   } catch (Exception ex) {
+   *   } catch (IOException ex) {
```

Comment:
The end `}` isn't aligned with the opening try(). Please fix the spacing. This should be done below as well.

Reply:
Done